elastic/kibana
# [Infrastructure UI] Metrics explanation tooltips

## Summary

This PR adds better tooltip content to the KPI tiles and table columns. It also adds links to hosts metrics documentation

![image](https://github.com/elastic/kibana/assets/2767137/3df60a99-32c0-4e08-9a84-e489db2d53fc)

The formulas displayed on the tooltips/popovers are the same ones used in the Lens charts. Even though the table content executes a query in elasticsearch, with pipeline aggregations, they can be translated into the equivalent Lens formulas.

### How to test this PR

- Start a local kibana instance
- Navigate to `Inventory > Hosts`
- Click on `What are these metrics?` links, they should redirect to https://www.elastic.co/guide/en/observability/current/host-metrics.html
- Hover over the KPI tiles and verify their content
- Click on `?` icons next to the table column label and verify their content
  - Clicking on `?` icon must not sort the table
